ostree-restore.sh

```diff
@@ -23,11 +23,10 @@ base_container=$(ostree cat backup /rpm-ostree.status | awk '/ostree-unverified-
 
 log_it Importing base OCI
 cp pull-secret.json /etc/ostree/auth.json
-ostree container unencapsulate --repo /ostree/repo $base_container --write-ref base
```

Comment:
Among other things, this avoids having a `base` ref that we now need to potentially GC later.